custom-login/src/Navbar.jsx

```diff
@@ -12,32 +12,38 @@
 
 import { useOktaAuth } from '@okta/okta-react';
 import React from 'react';
+import { useHistory, Link } from 'react-router-dom';
 import { Container, Icon, Image, Menu } from 'semantic-ui-react';
 
 const Navbar = () => {
-  const { authState, authService } = useOktaAuth();
+  const history = useHistory();
+  const { authState, oktaAuth } = useOktaAuth();
 
-  const login = async () => authService.login('/');
-  const logout = async () => authService.logout('/');
+  const login = async () => history.push('/login');
+  const logout = async () => oktaAuth.signOut();
 
   return (
     <div>
       <Menu fixed="top" inverted>
         <Container>
-          <Menu.Item as="a" header href="/">
+          <Menu.Item header>
             <Image size="mini" src="/react.svg" />
             &nbsp;
-            Okta-React Sample Project
+            <Link to="/">Okta-React Sample Project</Link>
           </Menu.Item>
           {authState.isAuthenticated && (
-          <Menu.Item id="messages-button" as="a" href="/messages">
+          <Menu.Item id="messages-button">
             <Icon name="mail outline" />
-            Messages
+            <Link to="/messages">Messages</Link>
           </Menu.Item>
           )}
-          {authState.isAuthenticated && <Menu.Item id="profile-button" as="a" href="/profile">Profile</Menu.Item>}
-          {authState.isAuthenticated && <Menu.Item id="logout-button" as="a" onClick={logout}>Logout</Menu.Item>}
-          {!authState.isPending && !authState.isAuthenticated && <Menu.Item as="a" onClick={login}>Login</Menu.Item>}
+          {authState.isAuthenticated && (
+            <Menu.Item id="profile-button">
+              <Link to="/profile">Profile</Link>
+            </Menu.Item>
+          )}
+          {authState.isAuthenticated && <Menu.Item id="logout-button" onClick={logout}>Logout</Menu.Item>}
+          {!authState.isPending && !authState.isAuthenticated && <Menu.Item onClick={login}>Login</Menu.Item>}
```

Comment:
Is the `!authState.isAuthenticated` redundant and distracting here?

Reply:
It's in Navbar which is not protected by SecureRoute. Probably still need to keep the logic here.

Reply:
But when will isAuthenticated be true if isPending is true?

Reply:
I think the logic here is to show `Login` button when not authenticated, so we'll need the `authState.isAuthenticated` flag. Also, we don't want to show the button when in pending state, so we use both.